democracy/migrations/0015_add_organization_model.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2016-05-03 09:49
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('democracy', '0014_non_editable_id'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Organization',
+            fields=[
+                ('created_at', models.DateTimeField(db_index=True, default=django.utils.timezone.now, editable=False, verbose_name='time of creation')),
+                ('modified_at', models.DateTimeField(default=django.utils.timezone.now, editable=False, verbose_name='time of last modification')),
+                ('published', models.BooleanField(db_index=True, default=True, verbose_name='public')),
+                ('deleted', models.BooleanField(db_index=True, default=False, editable=False, verbose_name='deleted')),
+                ('id', models.CharField(blank=True, help_text='You may leave this empty to automatically generate an ID', max_length=32, primary_key=True, serialize=False, verbose_name='identifier')),
```

Comment:
Actually this is a bit wrong because `StringIdBaseModel` has changed, I *think* correct would be `('id', models.CharField(editable=False, max_length=32, primary_key=True, serialize=False, verbose_name='identifier'))`